### PR TITLE
CPM-136: add user group exports

### PR DESCRIPTION
**Pull request by nmarniesse**

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml [hunk 1]
```diff
@@ -706,6 +706,23 @@ jobs:
...
+            enclosure: '"'
+            withHeader: true
+    xlsx_user_group_export:
+        connector: Akeneo XLSX Connector
+        alias: xlsx_user_group_export
+        label: Demo XLSX user group export
+        type: export
+        configuration:
+            withHeader: true
+            linesPerFile: 10000
```
Comment (mmetayer) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml:725]: No need to add the profiles to the catalogs IMO

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml [hunk 2]
```diff
@@ -120,6 +120,23 @@ jobs:
...
+            enclosure: '"'
+            withHeader: true
+    xlsx_user_group_export:
+        connector: Akeneo XLSX Connector
+        alias: xlsx_user_group_export
+        label: Demo XLSX user group export
+        type: export
+        configuration:
+            withHeader: true
+            linesPerFile: 10000
```
Comment (mmetayer) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml:139]: same (but it means you'll have to create the profile in your test)

File: src/Akeneo/UserManagement/Component/Normalizer/GroupNormalizer.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * @author    Nicolas Marniesse <nicolas.marniesse@akeneo.com>
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+final class GroupNormalizer implements NormalizerInterface, CacheableSupportsMethodInterface
+{
+    /** @var string[] */
+    protected array $supportedFormats = ['array', 'standard', 'internal_api'];
```
Comment (mmetayer) [inline src/Akeneo/UserManagement/Component/Normalizer/GroupNormalizer.php:19]: not internal api IMO; the internal API format would include a `meta`, like in https://github.com/akeneo/pim-community-dev/blob/379c42803da45e2dbf3422d797c91745f27bb677/src/Akeneo/UserManagement/Bundle/Controller/Rest/UserGroupController.php#L34-L47)

File: src/Akeneo/UserManagement/Bundle/Resources/config/normalizers.yml [hunk 4]
```diff
@@ -13,3 +13,8 @@ services:
             - []
         tags:
             - { name: pim_internal_api_serializer.normalizer }
+
+    pim_user.normalizer.user_group:
+        class: Akeneo\UserManagement\Component\Normalizer\GroupNormalizer
+        tags:
+            - { name: pim_internal_api_serializer.normalizer }
```
Comment (mmetayer) [inline src/Akeneo/UserManagement/Bundle/Resources/config/normalizers.yml:20]:
```suggestion
            - { name: pim_standard_format_serializer.normalizer }
```

**Review by mmetayer — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml (hunk 1 above)
Comment (nmarniesse) [reply src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml:725]: seen with our beloved PM

**Pull request merged**